ui/src/app/testing/fake-aggregation.service.ts

```diff
@@ -0,0 +1,97 @@
+import {AggregationResponse, JobStatus, TimeFrame} from "../shared/model/models";
+
+const aggregationResponse: AggregationResponse =
+  {
+   aggregations: [
+     {
+       entries: [
+         {
+           label: "owner1",
+           statusCounts: {
+             counts: [
+               {
+                 count: 2,
+                 status: JobStatus.Succeeded
+               },
+               {
+                 count: 1,
+                 status: JobStatus.Failed
+               }
+             ]
+           }
+         },
+         {
+           label: "owner2",
+           statusCounts: {
+             counts: [
+               {
+                 count: 4,
+                 status: JobStatus.Succeeded
+               },
+               {
+                 count: 6,
+                 status: JobStatus.Failed
+               }
+             ]
+           }
+         }
+       ],
+       key: "Owner"
```

Comment:
nit: seems like it would be more logical to put the key before the entries

Reply:
fixied.